mainnet-canister-revisions.json

```diff
@@ -63,6 +63,10 @@
     "rev": "250daf4dd0cf7ea74c496b45457dd47ced16368c",
     "sha256": "28df08368379862ee17f69542b0779670deec200a08f23540c3e6168c07cc9aa"
   },
+  "cycles_ledger": {
+    "sha256": "a2a0c65a94559aed373801a149bf4a31b176cb8cbabf77465eb25143ae880f37",
+    "tag": "cycles-ledger-v1.0.5"
```

Comment:
Idea for future work: I feel the current `mainnet-canister-revisions.json` and the logic that processes it in `bazel/mainnet-canisters.bzl` is a bit arbitrary in the sense that when there's a `rev` field we assume the WASM can be downloaded from https://download.dfinity.systems and when there's a `tag` field the WASM can be downloaded from GitHub.

A simplification would be to replace the `rev` and `tag` fields with a `url` field pointing directly to the WASM.

Reply:
Good point! https://dfinity.atlassian.net/browse/NNS1-3986